### DISCUSSION TITLE
Maintain schedules of velero after deletion and reinstallation

### DISCUSSION THREAD

**Discussion by junoriosity** [vmware-tanzu/velero#5209] [Community support Q&A]
I am currently installing `velero` via the usual `velero install ...` command.

I create the schedule

```
velero create schedule test-schedule --schedule="*/1 * * * *" --ttl 0h5m0s --include-namespaces wp
```

```
velero get schedules
NAME            STATUS    CREATED                          SCHEDULE      BACKUP TTL   LAST BACKUP   SELECTOR
test-schedule   Enabled   2022-08-14 12:46:25 +0200 CEST   */1 * * * *   5m0s         n/a           <none>
```

and it works

```
velero get backup
NAME                           STATUS      ERRORS   WARNINGS   CREATED                          EXPIRES   STORAGE LOCATION   SELECTOR
test-schedule-20220814104737   Completed   0        0          2022-08-14 12:47:37 +0200 CEST   4m        default            <none>
```

Now, when I delete the `velero` namespace and reinstall velero as outlined from above then the backups are still there, but there are no schedules left.

Is there a way to keep them?

**Comment by junoriosity** [comment]
@ncdc @aramprice @carlisia @dprotaso  Hi, do you have some input for me here?  😀

**Reply by junoriosity** [reply]
@ncdc @aramprice @carlisia @dprotaso Hi again, do you now have some input for me here?  😀

**Comment by dprotaso** [comment]
I don’t have access to this issue and I’ve never worked on velero so not
sure why I’m being pinged.
